File: client/src/app/model/event-dto.ts
export class EventDto {
  eventName:String;
  eventVenue:String;
  eventDate:String;
  eventDescription:String;
  eventPoster:string;

  constructor(eventName: String, eventVenue: String, eventDate: String, eventDescription: String,eventPoster:string) {
    this.eventName = eventName;
    this.eventVenue = eventVenue;
    this.eventDate = eventDate;
    this.eventDescription = eventDescription;
    this.eventPoster = eventPoster;
  }

}

File: server/src/main/java/com/upsintern/emplorium/dto/EventDto.java
package com.upsintern.emplorium.dto;


import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.sql.Timestamp;
import java.util.Date;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class EventDto {
    String eventName;
    String eventVenue;
    @JsonFormat(pattern = "dd-MM-yyyy HH:mm a")
    private Timestamp eventDate;
    String eventDescription;
    String eventPoster;
}
